docs/DeveloperGuide.md

```diff
@@ -328,44 +328,41 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 **MSS**
 1. User requests to add a contact.
 2. User specifies details of the contact.
-3. System adds the contact.
-
-Use case ends.
+3. AB3 adds the contact.<br>
+   Use case ends.
 
 **Extensions**
 
-2a. System detects user input is invalid. <br>
-&emsp;2a1. System displays and error message. <br>
-&emsp;2a2. User enters new input. <br>
-&emsp;Steps 2a1-2a2 are repeated until user input is valid. <br>
-&emsp;Use case resumes from step 2.
+<div class="step">2a. AB3 detects user input is invalid.</div>
+<div class="sub-step">2a1. AB3 displays an error message.</div>
+<div class="sub-step">2a2. User enters new input.</div>
+<div class="sub-step">Steps 2a1-2a2 are repeated until user input is valid.</div>
+<div class="sub-step">Use case resumes from step 2.</div>
```

Comment:
Should this be `Use case resumes from step 3.` instead?

Reply:
Fixed